File: src/baekjoon/step/hard1/PROB2941.java
package src.baekjoon;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class PROB2941 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String word = br.readLine();

        numnerOfAlphabet(word);

    }

    private static void numnerOfAlphabet(String word) {
        String[] alphabet = {"c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="};

        for (int i = 0; i < alphabet.length; i++) {
            if (word.contains(alphabet[i])) {
                word = word.replace(alphabet[i], "0");
            }
        }

        System.out.println(word.length());
    }

}

File: src/baekjoon/level/silver/PROB2941.py
# 단어가 몇개의 크로아티아 알파벳으로 이루어져 있는지 출력
# 아래는 특수한 형태의 알파벳이고, 나머지는 영어 알파벳과 같다.

croatia_alphabet = ["c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="]

word = input()

for i in croatia_alphabet:
    word = word.replace(i, '#')

print(len(word))
